contracts/SafeProtocolRegistry.sol

```diff
@@ -59,8 +59,8 @@ contract SafeProtocolRegistry is ISafeProtocolRegistry, Ownable2Step {
         ModuleInfo memory moduleInfo = listedModules[module];
 
         // Check if module is already listed or if moduleTypes is greater than 8.
-        // Maximum allowed value of moduleTypes is 7. i.e. 2^0 (Plugin) + 2^1 (Function Handler) + 2^2 (Hooks)
-        if (moduleInfo.listedAt != 0 || moduleTypes > 7) {
+        // Maximum allowed value of moduleTypes is 31. i.e. 2^0 (Plugin) + 2^1 (Function Handler) + 2^2 (Hooks) + 2^3 (Signature Validator hooks) + 2^4 (Signature Validator)
+        if (moduleInfo.listedAt != 0 || moduleTypes > 31) {
             revert CannotAddModule(module, moduleTypes);
```

Comment:
This error is not really descriptive for the check we do, right?

`moduleInfo.listedAt != 0` -> Module not available in the registry

`moduleTypes > 31` -> Invalid module types

Reply:
https://github.com/safe-global/safe-core-protocol/pull/109/commits/dbbafeb1938653760f3012fc889de29cfcc147b6